about.html

```diff
@@ -93,7 +94,25 @@ <h2 class="blackheader1">Collaboration</h2>
     <h2 class="together">Let's Work Together</h2>
 
     <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
-</div>
+  </div>
+
+<form>
+  <h2>Contact Us! </h2>
+  First name:<br>
+  <input type="text" name="firstname"><br>
+  Last name:<br>
+  <input type="text" name="lastname">
+<br><br>
+  <input type="submit" value="Submit">
+  <br>
+  <br>
+</form>
+
+
```

Comment:
This could also go into another page so you could link to it using the contact link on the navigation